src/js/settings.js

```diff
@@ -0,0 +1,77 @@
+export function saveLanguageChoice() {
+	// does not change the language atm but the value is saved
+	const languageSelector = document.getElementById('languageSelector');
+
+	let settingsJSON = JSON.parse(localStorage.getItem('settingsJSON')) || {};
+
+	if (!settingsJSON.langSelected) {
+		settingsJSON.langSelected = 'english';
+		localStorage.setItem('settingsJSON', JSON.stringify(settingsJSON));
+	}
+
+	if (settingsJSON.langSelected) {
+		languageSelector.value = settingsJSON.langSelected;
+	}
+
+	console.log(settingsJSON.langSelected);
+
+	languageSelector.onchange = function () {
```

Comment:
Pune te rog cele 3 functii cu **.onchange** intr-o alta functie numita **addInputsChangeEvents** si pe aia o apelezi aici in settings.js. Astfel adaugi listeners pentru change. Iar celelalte 3 functii le lasi doar pentru a salva pe local storage.